### ISSUE TITLE
debug command line error

### ISSUE THREAD

**Issue by wanggaa** [microsoft/vscode#193079]
Type: <b>Bug</b>

make any breakpoint
start debug(F5)
restart debug(ctrl + shift + f5)
use debug prompt(ctrl + shift + y)
type any thing, there is no response, only if you change the prompt to any other channel(as terminal) and then change back to debug prompt

VS Code version: Code 1.82.1 (6509174151d557a81c9d0b5f8a5a1e9274db5585, 2023-09-08T08:45:05.575Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i7-13700K (24 x 3418)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.70GB (50.23GB free)|
|Process Argv|--crash-reporter-id 318d26e1-f54a-4e4f-b7fa-c32f04350a17|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-22.04|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|13th Gen Intel(R) Core(TM) i7-13700K (24 x 3417)|
|Memory (System)|31.19GB (29.58GB free)|
|VM|0%|
</details><details><summary>Extensions (27)</summary>

Extension|Author (truncated)|Version
---|---|---
protein-viewer|Ari|0.1.0
python-environment-manager|don|1.2.2
python-extension-pack|don|1.7.0
gitlens|eam|14.3.0
vscode-test-explorer|hbe|2.21.1
vsc-python-indent|Kev|1.18.0
vscode-python-test-adapter|lit|0.7.1
vscode-docker|ms-|1.26.0
vscode-language-pack-zh-hans|MS-|1.82.2023091309
python|ms-|2023.16.0
vscode-pylance|ms-|2023.9.10
jupyter|ms-|2023.8.1002501831
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.309.0
remote-ssh|ms-|0.106.4
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.3
vscode-remote-extensionpack|ms-|0.24.0
remote-explorer|ms-|0.4.1
remote-server|ms-|1.4.3
test-adapter-converter|ms-|0.1.8
autodocstring|njp|0.6.1
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
282f8724:30602487
f6dab269:30613381
showlangstatbar:30737416
0bi6i642:30835152
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
asynctok:30821568
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30821572
dsvsc015:30829745

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate, and will be fixed in upcoming 1.82.2

**Comment by roblourens** [comment]
https://github.com/microsoft/vscode/issues/192653
